tests/System.Buffers.Experimental.Tests/MemoryTests.cs

```diff
@@ -1,14 +1,18 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using System.Buffers;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
 using Xunit;
 
-namespace System.Slices.Tests
+namespace System.Buffers.Tests
```

Comment:
Is it ok to have the same namespace and class name in different assemblies?

- System.Buffers.Experimental.Tests contains System.Buffers.Tests.MemoryTests
- System.Buffers.Primitives.Tests contains System.Buffers.Tests.MemoryTests